### PR TITLE
Bulleted list of features the app has on README

### LINKED ISSUE #528: Create a bulleted list of features the app has on the README
## Description
As a contributor,
I need to know the app's features,
so that I can know more about the app before working on it.

The README has practical information on how to contribute to the app and a small description of the app, but misses some information about the features it has, like see members of the app, send mentorship request.

### Update [Required]
- [ ] Add a bullet list of high-level features to the README

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
1 hour

### PR DESCRIPTION
**Pull request by madhhuurrii**
### Description

Fixes #528 
 Created a bulleted list of features the mentorship App has on the README Page.

### Type of Change:

- Documentation

### How Has This Been Tested?

image
image

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by madhhuurrii** [comment]
Hii @isabelcosta, would u please help me in unsuccessful checks, how to solve it ?

File: jsr [hunk 1]
```diff
@@ -0,0 +1,161 @@
+See 'git help <command>' to read about a specific subcommand
```
Comment (isabelcosta) [inline jsr]: can you please remove this file as it is not related to the issue you are solving here.

File: README.md [hunk 2]
```diff
@@ -27,6 +28,25 @@ This API is being used by 3 frontend projects currently being developed:
 - [Contact](#contact)
 - [License](#license)
 
+## Features
+
+- Once the App is installed user can view an onboarding screen which introduces the app and give user an idea of how it works.
```
Comment (isabelcosta) [inline README.md]: there are few bullet points here that are unrelated to what the backend code provides, like this line here. This is only related with the android repository. Can you please remove this, and other bullet points which are unrelated, since they might be only implemented on the frontend app implementations.

File: README.md [hunk 3]
```diff
@@ -27,6 +28,25 @@ This API is being used by 3 frontend projects currently being developed:
...
+- A member gets an option to be a Mentor, Mentee or Both.
+- A member can build/customise the app profile with username, bio, skills, interests, location, occupation, etc. anytime.
+- A member can read and know more about Mentorship System including the terms and conditions, privacy and policy and code of conduct.
+- A member can directly access the AnitaB-org Github repo, Zulip chat and website from this App.
+- A member can refresh the every page.
```
Comment (isabelcosta) [inline README.md]: these bullet points could be removed since they are unrelated to the backend code

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@madhhuurrii thank you for this contribution 🎉 
I left some change requests - I see some bullet point features that are unrelated to the backend GitHub repository so they don't need to be in this README. I think you could reduce these bullet points to 5 and be enough :)

If you take a quick look at the Swagger UI definition, you have an idea of what is possible to do at this point - https://mentorship-backend-temp.herokuapp.com/.

Let me know if you have any doubts :)

File: README.md (hunk 3 above)
Comment (madhhuurrii) [reply README.md]: Sure will do it!

**Comment by madhhuurrii** [comment]
Hii @isabelcosta I have made appropriate changes Please review

**Comment by madhhuurrii** [comment]
Hii @isabelcosta @vj-codes please review my PR

**Review by devkapilbansal — APPROVED** [review]
LGTM :+1:

File: README.md [hunk 4]
```diff
@@ -27,6 +28,13 @@ This API is being used by 3 frontend projects currently being developed:
 - [Contact](#contact)
 - [License](#license)
 
+## Features
+
+- Mentorship System is an application that matches women in tech to mentor each other, in career developmemnt, through 1:1 relations during a certain period of time.
```
Comment (isabelcosta) [inline README.md:33]: this is already up in the top of the README so it could be removed from here.

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@madhhuurrii you removed some interesting content here. You were close to a holistic way of defining the features :) Here's what I would suggest. If you look at what the API provides, as feature. What can be used in a frontend. You have 3 main APIs, and then you can look at the endpoints to identify the "sub-features".
The backend has endpoints to:
- Login and register a new user
- Create a user profile, view and edit user profiles
- Change user password, refresh jwt tokens
- Create relation between two users with a fixed period of time
- Both mentor and mentee in a relation can create tasks
- Assign and remove admin roles to users and list all admins
- etc

These can give an idea to a developer that finds the project, know what capabilities this has at the moment, and in this way see what could be missing.

Could you adjust the features list to look a little more like this ^^?

**Comment by madhhuurrii** [comment]
Will do it

**Comment by vj-codes** [comment]
@madhhuurrii hey the screenshot that you've added in the PR description is exactly same as the features section in the mentorship-android repo .
As suggested above , please update with the features related to this repo (mentorship-backend) , then this PR will be reviewed again:)
Thanks!

**Comment by vj-codes** [comment]
@madhhuurrii any updates?

**Comment by madhhuurrii** [comment]
> @madhhuurrii any updates?

Sorry, I'll be doing the changes in coming days

**Comment by vj-codes** [comment]
Closing due to inactivity
@madhhuurrii thank you for your contribution :)
